### PR TITLE
Jaclyn javascript 102

### PR DESCRIPTION
**Pull request by jkarnowski**

### CODE REVIEW

File: js102/index.html [hunk 1]
```diff
@@ -55,21 +55,78 @@
...
+
+				<!-- Overview of learning objectives -->
+				<section>
+					<h2>What You'll Be Learning About</h2>
+					<ul>
+						<li>Different ways to represent and manipulate data</li>
+						<li>Loops and Functions</li>
+						<li>Scope in JS</li>
+					</ul>
+					<p>Want to code along? Open up repl.it or run your js file in node</p>
```
Comment (brendajin) [inline js102/index.html]: the students might not know what node.js is at this point

File: js102/index.html [hunk 2]
```diff
@@ -93,12 +152,15 @@
...
+
 var i = 0;
-classes[i]; // ??
+
+classes[i]; //what gets returned?
 
 classes[1];
 
-classes.pop(); //??
+classes[4] = "HTML 201"
```
Comment (brendajin) [inline js102/index.html]: add a semicolon here for consistency

File: js102/index.html [hunk 3]
```diff
@@ -318,8 +390,37 @@
 				</section>
 				<!-- /Nested Objects -->
 
+				<!-- Functions Intro -->
+				<section>
+					<h2>So, what about Functions?</h2>
+					<ul>
+						<li>first-class object because they have properties and methods</li>
```
Comment (brendajin) [inline js102/index.html]: Can you explain what first-class means? This might be a bit jargon-y if we don't break it down

File: js102/index.html [hunk 4]
```diff
@@ -402,6 +503,7 @@
 };
 
 addTwo(3, 10); // 13	
+addTwo(4,5,6); //WHOA - what's happening here?
```
Comment (brendajin) [inline js102/index.html]: spaces after commas, please!

**Comment by brendajin** [comment]
@jkarnowski Great changes! I had a few minor points of feedback. I'll merge the PR for the workshop today but would love if you have a chance to update later :+1:

File: js102/index.html [hunk 5]
```diff
@@ -55,21 +55,78 @@
 					<p>Slides: <a href="http://yonet.github.io/JS102/#/" target="_blank">bit.ly/JS102</a></p>
 				</section>
 				<!-- /Welcome -->
+
+				<!-- Tone Setting -->
+				<section>
+					<h2>Hello!</h2>
+					<p>Some 'guidelines' for today</p>
+					<ul>
+						<li>We are here to support your learning in a safe, inclusive environment</li>
```
Comment (brendajin) [inline js102/index.html]: I really like these guidelines and wonder if we should update them in the intro slides

Comment (jkarnowski) [inline js102/index.html]: THANK YOU for great feedback and catching those syntax mishaps. Let's catch
up soon!

On Sun, Mar 6, 2016 at 10:30 AM Brenda Jin <notifications@github.com> wrote:

> In js102/index.html
> <https://github.com/gdisf/teaching-materials/pull/198#discussion_r55145571>
> :
>
> > @@ -55,21 +55,78 @@
> >  					<p>Slides: <a href="http://yonet.github.io/JS102/#/" target="_blank">bit.ly/JS102</a></p>
> >  				</section>
> >  				
> > +
> > +				
> > +				<section>
> > +					<h2>Hello!</h2>
> > +					<p>Some 'guidelines' for today</p>
> > +					<ul>
> > +						<li>We are here to support your learning in a safe, inclusive environment</li>
>
> I really like these guidelines and wonder if we should update them in the
> intro slides
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gdisf/teaching-materials/pull/198/files#r55145571>.
>
-- 
Jaclyn Karnowski

freelance web developer
co-founder, EdTalks

**Pull request merged**